src/lib/mailer/logNotifications.js

```diff
@@ -66,7 +66,8 @@ export default async function logEmailNotification(job, success, result) {
         template = path.resolve(emailTemplatePath, data.templatePath, 'subject.pug');
         break;
       default:
-        break;
+        logger.error(`Unknown job name: ${job.name}`);
+        throw new Error(`Unknown job name: ${job.name}`);
```

Comment:
do we want to throw an error in this case? why didn't we before?

Reply:
Krys made this change, seems like a good idea to throw an error when the job is unexpected